python/variable.cpp

```diff
@@ -184,17 +184,31 @@ of variances.)");
       .def(
           "__radd__", [](Variable &a, double &b) { return a + b * units::one; },
           py::is_operator())
+      .def(
+          "__radd__", [](Variable &a, int &b) { return a + b * units::one; },
+          py::is_operator())
```

Comment:
Should be `int64_t` to match the Python test? But may be correct actually, if it matches the Python `int` behavior.